third_party/2and3/dateutil/parser.pyi

```diff
@@ -33,6 +33,8 @@ class parser(object):
               ignoretz: bool = ..., tzinfos: Optional[Mapping[Text, tzinfo]] = ...,
               **kwargs: Any) -> datetime: ...
 
+def isoparse(dt_str: str) -> datetime: ...
```

Comment:
This can also take an `IO[str]`:
```
In [689]: dateutil.parser.isoparse(StringIO("2019"))
Out[689]: datetime.datetime(2019, 1, 1, 0, 0)
```

Reply:
From looking at the implementation, this can take also take `bytes` and byte readers.

https://github.com/dateutil/dateutil/blob/25eb8646da42528a6d1f057a9eef0fac42013077/dateutil/parser/isoparser.py#L22-L39